### DISCUSSION TITLE
Probable mistake percentage of percentage?

### DISCUSSION THREAD

**Discussion by Gounlaf** [dlemstra/Magick.NET#1547] [General] [answered]
:wave: 

Looking at:
https://github.com/dlemstra/Magick.NET/blob/167add42cd0875318d6bea71a8ebc14d6bbbb305/src/Magick.NET/MagickImage.cs#L1474-L1483

Seeing that `lower` & `upper` are divided again by 100 make me thing it's a mistake, but not sure about it.

Regards.

Edit: I see the same one for `DrawableFillOpacity.Draw`, and that's all (except for the `PercentageHelper` but what I understand is legit for this class).
And looking at ImageMagick corresponding functions add more confusion... I see some method that divide the percentage / 100 again, and some doesn't.

**Comment by dlemstra** [comment, marked as answer]
There is some difference between the methods in ImageMagick. This method requires a value from `0.0` to `1.0`.

**Reply by Gounlaf** [reply]
OK. Thanks for the clarification 🙂.
